canonical-web-and-design/vanilla-framework
# Accessibility: Misc markup fixes

## Done

- Remove type from `script` tags
- Add missing quotes to image src attr values
- Remove contentinfo role on footer tag
- Remove checked attr from select

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/876, 
https://github.com/canonical-web-and-design/vanilla-squad/issues/875,
https://github.com/canonical-web-and-design/vanilla-squad/issues/873


## QA

- Pull code
- Run `./run`
- Verify errors identified in theissues above have been resolved
